pkgs/development/python-modules/lazr-restfulclient/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, distro, httplib2, oauthlib, setuptools, six, wadllib }:
+
+buildPythonPackage rec {
+  pname = "lazr.restfulclient";
+  version = "0.14.3";
```

```suggestion
  version = "0.14.3";
  disabled = isPy27; # namespace is broken for python2
```

python2 namespace is broken, they follow pep420, and python2 is unable to parse it.

```
builder for '/nix/store/xxqjk25awhk4r6g6wh30m7x02g5y82lk-python2.7-lazr.restfulclient-0.14.3.drv' failed with exit code 1; last 10 log lines:
  pythonRemoveBinBytecodePhase
  pythonImportsCheckPhase
  Executing pythonImportsCheckPhase
  Check whether the following modules can be imported: lazr.restfulclient
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "<string>", line 1, in <lambda>
    File "/nix/store/bh447min7f6vqdw3x1iywr1vxyx4ix8y-python-2.7.17/lib/python2.7/importlib/__init__.py", line 37, in import_module
      __import__(name)
  ImportError: No module named lazr.restfulclient
```

---

Fixed, thanks!